xbmc/platform/darwin/osx/XBMCApplication.mm

```diff
@@ -346,42 +301,40 @@ - (NSMenu*)applicationDockMenu:(NSApplication*)sender
 
 @end
 
-int main(int argc, char* argv[])
+int main(int argc, const char* argv[])
 {
+#if defined(DEBUG) && 0 // SHH enable as needed
+  struct rlimit rlim;
+  rlim.rlim_cur = rlim.rlim_max = RLIM_INFINITY;
+  if (setrlimit(RLIMIT_CORE, &rlim) == -1)
+    CLog::Log(LOGDEBUG, "Failed to set core size limit ({})", strerror(errno));
+#endif
+
   @autoreleasepool
   {
     /* Copy the arguments into a global variable */
     /* This is passed if we are launched by double-clicking */
     if (argc >= 2 && strncmp(argv[1], "-psn", 4) == 0)
     {
-      gArgv = (char**)malloc(sizeof(char*) * 2);
+      gArgv = (const char**)malloc(sizeof(char*) * 2);
```

Comment:
C style cast, usually we use these opportunities to change them (if possible)

Reply:
well, all right then :-)